tests/foreman/cli/test_satellitesync.py

```diff
@@ -383,6 +400,92 @@ def test_positive_complete_library_rh_repo(
             != ''
         )
 
+    @pytest.mark.tier3
+    @pytest.mark.upgrade
+    def test_positive_export_repository_docker(
+        self, target_sat, export_import_cleanup_module, module_org, docker_repo
+    ):
+        """Export docker repo via complete and incremental repository.
+
+        :id: 3c666ffd-d287-4006-b3a0-66d892fe4250
+
+        :setup:
+            1. Have a synchronized docker-type repo with immediate download policy.
+
+        :steps:
+            1. Export complete repository.
+            2. Export incremental repository.
+
+        :expectedresults:
+            1. Export path is created (with expected files) for complete export.
+            2. Export path is created (with expected files) for incremental export.
+
+        :BZ: 1650468
+
+        :customerscenario: true
+        """
+        # Verify export directory is empty
+        assert target_sat.validate_pulp_filepath(module_org, PULP_EXPORT_DIR) == ''
+        # Export complete and check the export directory
+        ContentExport.completeRepository({'id': docker_repo['id']})
```

Comment:
Instead of importing any of these CLI classes we should be able to access them through `target_sat.cli`

Reply:
Sure, I just went with the current module style. I can update it.